react-native/WINDOWS.md

```diff
@@ -0,0 +1,47 @@
+# Installation
+
+### Requirements
+
+1. Node
+1. Android Studio.
+
+### Installing Node
+
+1. [Download Node](https://nodejs.org/en/download/) (choose Node version LTS/Current).
```

Comment:
Does `npm` come with `node`?

Reply:
@Satendra-SR Yes, it comes with the node